### ISSUE TITLE
Website link disappears

### ISSUE THREAD

**Issue by chakrabortydeepro** [PanditProject/Website#38]
[This](https://www.bharatiyakritisampada.nic.in/) entry doesn't display the website link even though the website link was provided in the input.

**Comment by chakrabortydeepro** [comment]
Sorry, Amir. Here is the link: [https://www.panditproject.org/entity/109736/print](https://www.panditproject.org/entity/109736/print)

**Comment by AmirSimantov** [comment]
Hi Deepro, this is a Print entity, and it doesn't have a "Website" field. Have you meant something else?

**Comment by AmirSimantov** [comment]
Yes, you were correct! I have fixed the bug; please let me know if it works for you now.
Please note the field's name is "Web page URL" and not "Website"; it confused me :)
